llvm/include/llvm/IR/Type.h

```diff
@@ -339,6 +357,9 @@ class Type {
   /// ppc long double), this method returns -1.
   int getFPMantissaWidth() const;
 
+  /// Return the width of the mantissa of a decimal floating-point type.
+  int getDFPMantissaWidth() const;
+
```

Comment:
Let's clarify the documentation for both `getFPMantissaWidth()` and `getDFPMantissaWidth()`. For the former, the value returned is the precision in bits. For the latter, the value is the precision in digits. The X.2.1, "Interchange floating types" section of [WG14 N2601](https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2601.pdf) specifies these values.

In order to avoid confusion, I think we should name these functions differently. Joshua has requested that we not rename `getFPMantissaWidth()` now and that is reasonable. Perhaps name the new function `getDFPPrecisionInDigits()`?